go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/RangelReale/osincli v0.0.0-20160924135400-fababb0555f2
 	github.com/apparentlymart/go-cidr v1.1.0
 	github.com/davecgh/go-spew v1.1.1
-	github.com/docker/distribution v2.8.1+incompatible
+	github.com/distribution/distribution/v3 v3.0.0-20230530204932-ba46c769b3d1
```

Comment:
I'd expect at least library-go be updated along this one, to match with the one that has docker distribution v3.

Reply:
gave a stab at it, let's see what breaks.

Reply:
Looks like all of the odd unit test failures I was seeing locally are not seen by CI 🎉